Makefile

```diff
@@ -1707,13 +1705,9 @@ dataplane_kuttl_prep: dataplane_kuttl_cleanup input ansibleee infra baremetal da
 	devsetup/scripts/gen-ansibleee-ssh-key.sh
 	mkdir -p ${OPERATOR_BASE_DIR} ${OPERATOR_DIR}
 	pushd ${OPERATOR_BASE_DIR} && git clone ${GIT_CLONE_OPTS} $(if $(DATAPLANE_BRANCH),-b ${DATAPLANE_BRANCH}) ${DATAPLANE_REPO} "${OPERATOR_NAME}-operator" && popd
-	$(eval $(call vars,$@,ansibleee))
-	make wait
-	$(eval $(call vars,$@,infra))
-	make wait
-	$(eval $(call vars,$@,openstack-baremetal))
-	make wait
-	$(eval $(call vars,$@,dataplane))
+	make wait OPERATOR_NAME=ansibleee
```

Comment:
I think this has broken the dataplane kuttl jobs. https://github.com/openstack-k8s-operators/install_yamls/pull/711